dojo/importers/importer/importer.py

```diff
@@ -0,0 +1,276 @@
+import datetime
+from dojo.importers import utils as importer_utils
+from dojo.models import Test, Finding, \
+    Test_Type, \
+    Endpoint, BurpRawRequestResponse, \
+    Endpoint_Status, \
+    Test_Import
+
+from dojo.utils import get_current_user, max_safe
+
+from django.core.exceptions import MultipleObjectsReturned
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+import dojo.notifications.helper as notifications_helper
+import base64
+import logging
+
+logger = logging.getLogger(__name__)
+deduplicationLogger = logging.getLogger("dojo.specific-loggers.deduplication")
+
+
+class DojoDefaultImporter(object):
+
+    def create_test(self, scan_type, engagement, lead, environment, tags=None,
+                    scan_date=None, version=None, branch_tag=None, build_id=None, commit_hash=None, now=timezone.now()):
+
+        test_type, created = Test_Type.objects.get_or_create(
+            name=scan_type)
+
+        if created:
+            logger.info('Created new Test_Type with name %s because a report is being imported', test_type.name)
+
+        test = Test(
+            engagement=engagement,
+            lead=lead,
+            test_type=test_type,
+            target_start=scan_date if scan_date else now.date(),
+            target_end=scan_date if scan_date else now.date(),
+            environment=environment,
+            percent_complete=100,
+            version=version,
+            branch_tag=branch_tag,
+            build_id=build_id,
+            commit_hash=commit_hash,
+            tags=tags)
+        try:
+            # TODO What is going on here?
+            test.full_clean()
+        except ValidationError:
+            pass
+
+        test.save()
+        return test
+
+    def process_parsed_findings(self, test, parsed_findings, scan_type, user, active, verified, minimum_severity=None,
+                                endpoints_to_add=None, push_to_jira=None, now=timezone.now()):
+        logger.debug('endpoints_to_add: %s', endpoints_to_add)
+        new_findings = []
+        items = parsed_findings
+        logger.debug('starting import of %i items.', len(items) if items else 0)
+        i = 0
+        for item in items:
+            sev = item.severity
```

Comment:
please remove the complete bloc.
I want to enforce by unit tests that what's coming from parsers are ok better than relying on hack like this one in the upper levels